### PR TITLE
Feat/boarding rework

### PR DESCRIPTION
**Pull request by joaoqueiroz95**
[Uploading new-deboard.webm…]()

### CODE REVIEW

File: src/screens/Boarding.tsx [hunk 1]
```diff
@@ -34,57 +40,88 @@ export function BoardingScreen() {
...
-              size="lg"
-              color={theme.colors.primary.contrastText}
-              style={styles.buttonText}>
-              {t("boarding.send-qr-scan")}
+    <>
+      <DeboardModal isOpen={modalOpen} onCancel={handleCloseModal} onConfirm={handleConfirmModal} />
+      <ScreenLayout
+        isScrollable
+        headerProps={{
+          title: "Boarding",
```
Comment (joel-gouveia) [inline src/screens/Boarding.tsx:72]: i8

File: src/screens/Boarding.tsx [hunk 2]
```diff
@@ -34,57 +40,88 @@ export function BoardingScreen() {
...
+          icon: <Icon name="sailboat" size={24} color={theme.colors.primary.main} />,
+        }}
+        footerButtons={[
+          {
+            subtitle: "Main Menu",
```
Comment (joel-gouveia) [inline src/screens/Boarding.tsx:77]: i8

**Review by joel-gouveia — CHANGES_REQUESTED** [review]
Small changes but looks good

**Review by mathvaillant — APPROVED** [review]

**Review by joel-gouveia — APPROVED** [review]
LGTM!

**Pull request merged**